CHANGELOG.md

```diff
@@ -11,7 +11,30 @@ Changes since the last release
 
 ### Changed defaults / behaviours
 
+-   Example
+
+### Deprecated / removed options and commands
+
+### Security Related Fixes
+
+### Bug Fixes
+
+### Testing / Development
+
+### Known Issues
+
+## v3.10.16 \[2025-09-08\]
+
+Bug fix release. Mostly cvmfs and condor job wrapper related.
+
+### New features / functionalities
+
+-   Create_cvmfsexec_distros.sh updated to default to EL9 and improved command syntax (PR #582)
```

Comment:
Can you clarify what you mean by "updated to default to EL9"? Do you mean addition of EL9 to the list of platforms available?